web_external/Datasets/datasetInfoWidget.pug

```diff
@@ -11,6 +11,13 @@ table.table.table-condensed.isic-dataset-info-table
     tr
       td.isic-dataset-info-table-label License
       td= dataset.get('license')
+    tr
+      td.isic-dataset-info-table-label Attribution
+      td= dataset.get('attribution')
```

Comment:
Attribution isn't available for the dataset summary, so displaying it should be inside a conditional.